app/views/nsm/steps/view_claim/calls.html.erb

```diff
@@ -1,5 +1,5 @@
 <% title t('.page_title') %>
-<% decision_step_header(path: nsm_steps_view_claim_path(current_application.id, section: params.fetch(:section, 'claimed_costs'), anchor: 'letters-and-calls-tab')) %>
+<% decision_step_header(path: :back) %>
```

Comment:
Investigate why this is a fixed path

Reply:
This was introduced in [this](https://github.com/ministryofjustice/laa-submit-crime-forms/commit/3d3e43dd09139e246930411a015c550269ba4ba3#diff-7909a93b507473f87db01fffc27ee7a7f51d49668c8245a6b317ac8fcaccb602) commit without an obvious indicator as to why other than it seeming related to turbo frames.

Will ask David when he's back

Reply:
David confirmed this was fine to change